UPGRADE-6.0.md

```diff
@@ -21,6 +21,7 @@ Form
 
  * Added the `getIsEmptyCallback()` method to the `FormConfigInterface`.
  * Added the `setIsEmptyCallback()` method to the `FormConfigBuilderInterface`.
+ * Added a third `callable|null $filter` argument to `ChoiceListFactoryInterface::createListFromChoices` and `ChoiceListFactoryInterface::createListFromLoader` methods.
```

Comment:
Added argument `callable|null $filter` to `ChoiceListFactoryInterface::createListFromChoices()` and `createListFromLoader()`